dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -358,47 +402,10 @@ def get_job_log_types(self, job):
 
         return res
 
-    def get_job_output(self, job_name, job):
-        res_lines = []
+    def parse_query_result(self, res_lines):
         schema = []
         rows = []
 
-        no_of_retries = 0
-
-        while (
-            len(res_lines) <= MIN_LINES_TO_PARSE
```

Comment:
We don't need this logic.  checking for MIN_LINES_TO_PARSE(3) doesn't necessarily guarantee us that the job output log is complete.